scripts/helpers/wallet-precalculation.helper.js

```diff
@@ -7,7 +7,9 @@
 
 import { promises as fs } from 'fs';
 import { addressUtils, walletUtils } from '@hathor/wallet-lib';
-import config from '../../__tests__/integration/configuration/config-fixture';
+import settings from '../../__tests__/integration/configuration/settings-fixture';
```

Comment:
Is this file related to tests?

Reply:
Yes, the precalculation helps creating the seeds and precalculated addresses that are used on the tests.
The addresses are pre-calculated because the address derivation is somewhat cpu intensive and was hindering the tests execution times, so we just use a file with all addresses pre-calculated.

If we require more in the future or we want to re-generate the addresses, this file will be used.
The helper also expose method to get the pre-calculated wallet so some tests import it directly.

Reply:
Some comments here:
- The address derivation actually becomes orders of magnitude more cpu intensive _when running inside the test framework_, not sure why, but this was a good solution.
- I think this file belongs to the `scripts` folder because there may be other cases where it's useful to generate multiple wallets and their addresses in batch
- I agree that it feels weird having a script importing modules from the `tests` module. This indeed deserves more thought to come up with a better organization.